docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,87 @@
+image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]
+
+.Requirements
+[sidebar]
+--
+You must have _one_ of the following installed on the hosts you want to monitor:
+
+* *{agent}* - Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] and ensure the agent's {fleet-guide}/view-elastic-agent-status.html[status] must be `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
+* *{filebeat}* - Install {filebeat-ref}/filebeat-installation-configuration.html[{filebeat}] version 8.x. Earlier {filebeat} versions are incompatible with ECS and will prevent indicator data from displaying in the Indicators table.
+--
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+Threat intelligence is a research function that helps an organization make informed security decisions and actions. An organization's security operations center (SOC) team usually manages threat intelligence, which helps them understand current and emerging threats and recommend actions that will prevent, stop, and mitigate security threats.
+
+An indicator, also referred to as an indicator of compromise (IoC), is a piece of information associated with a known threat or reported vulnerability. There are many types of indicators, including URLs, files, domains, email addresses, and more. Within SOC teams, threat intelligence analysts use indicators to detect, assess, and respond to threats.
+
+[discrete]
+[[setup-intelligence-page]]
+== Set up the Intelligence page
+
+Install threat intelligence integrations to add indicators to the Intelligence page.
+
+NOTE: If you've already installed threat intelligence integrations, you do not need to complete the following steps. Indicator data will automatically display within the Indicators table.
+
+. Choose one of the following:
+* From the {security-app} main menu, click *Intelligence* -> *Add Integrations*.
+* From the {kib} main menu, click *Add integrations*. Scroll down the list of integration categories and select *Threat Intelligence* to filter for threat intelligence integrations.
+. Select a threat intelligence integration, then complete the integration's guided installation.
++
+NOTE: For more information about available fields, go to the https://docs.elastic.co/integrations[Elastic integration documentation] and search for a specific threat intelligence integration.
+
+. Return to the Intelligence page in {elastic-sec}. Refresh the page if indicator data isn't displaying.
+
+[discrete]
+[[troubleshoot-intel-page]]
+=== Troubleshooting
+If indicator data is not appearing in the Indicators table after you installed threat intelligence integrations:
+
+* Verify that the index holding indicator documents is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index holding indicator documents will differ based the way you're collecting indicator data:
```

Comment:
I feel like `index holding` needs a hyphen? `index-holding` 🤔 but feel free to omit if I'm wrong. 
```suggestion
* Verify that the index-holding indicator documents are included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index-holding indicator documents will differ depending on how you're collecting indicator data:
```

Reply:
Is this better/more clear, @jmikell821 :

* Verify that the indicators' index is included in the <<update-sec-indices,default {elastic-sec} indices>> (`securitySolution:defaultIndex`). The index will differ depending on how you're collecting indicator data: